backend-models/functional-requirements/FRD_resultedservice.md

```diff
@@ -1,21 +1,25 @@
 | Functionality requirement No. | Function Requirement Description |
-| FR1 | Student shall be able to view a submission |
-| FR2 | Teacher shall be able to filter dugga |
+| ------------- | ------------- |
+| FR1 | Teacher/super-user shall be able to view all student results |
+| FR2 | Teacher/super-user shall be able to filter student results |
 
 | FN No| Test Prosoal: |
-| FR1| Try to make a submission to see if it shows correct informations |
-| FR2| Try to filter a dugga |
+| ------------- | ------------- |
+| FR1| Login as different teachers/super-users and see if all can access the information |
+| FR2| Try to filter the student results |
```

Comment:
Tests that is focused more on data and that correct values are sent. for example: 
"Filter the student results and compare the result with database result with the values suppose to show"

Reply:
The tests are now more data focused